src/aquarium.py

```diff
@@ -39,12 +39,17 @@
     local,
     devices,
     nfs,
+    auth,
+    user,
 )
 
 logger: logging.Logger = fastapi_logger
 
 
 async def aquarium_startup(aquarium_app: FastAPI, aquarium_api: FastAPI):
+    import pydevd_pycharm
+    pydevd_pycharm.settrace('192.168.178.14', port=22222, stdoutToServer=True, stderrToServer=True)
```

Comment:
This should be conditional, somehow. Maybe passing an environment variable or something.

Reply:
Also, this is a dependency that is not being added. Which I support; I don't think adding this dependency is something we want. So maybe do this if the dependency exists, like in a `try` block of the sorts of

```
try:
  import pydevd_pycharm
  pydevd_pycharm.settrace(...)
except ModuleNotFound:
  pass
```

Reply:
This will be removed in the final PR.